common/src/androidTest/java/com/microsoft/identity/common/internal/platform/DevicePoPManagerTests.java

```diff
@@ -449,4 +451,22 @@ public void testGenerateShr() throws ClientException, MalformedURLException, Par
         Assert.assertNotNull(jwtClaimsSet.getClaim("cnf"));
         Assert.assertEquals(clientClaims, jwtClaimsSet.getClaim("client_claims"));
     }
+
+    @Test
+    public void testHasCertificateChain24() throws ClientException {
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.N) {
```

Comment:
Can you annotate this method and have that work?

Reply:
Annotated - I can revert it if this isn't respected by the test runner